administrator/language/de-DE/plg_system_stats.ini

```diff
@@ -5,24 +5,24 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_SYSTEM_STATS="System - Joomla!-Statistikerhebung"
-PLG_SYSTEM_STATS_BTN_NEVER_SEND="Niemals"
-PLG_SYSTEM_STATS_BTN_SEND_ALWAYS="Immer"
+PLG_SYSTEM_STATS_BTN_NEVER_SEND="Nein"
+PLG_SYSTEM_STATS_BTN_SEND_ALWAYS="Ja, Joomla! helfen"
 PLG_SYSTEM_STATS_BTN_SEND_NOW="Einmalig"
 PLG_SYSTEM_STATS_INTERVAL_LABEL="Intervall (Stunden)"
 PLG_SYSTEM_STATS_LABEL_CMS_VERSION="CMS-Version"
 PLG_SYSTEM_STATS_LABEL_DB_TYPE="DB-Typ"
 PLG_SYSTEM_STATS_LABEL_DB_VERSION="DB-Version"
-PLG_SYSTEM_STATS_LABEL_MESSAGE_TITLE="Joomla! bittet um Erlaubnis einige Basisstatistiken zu erheben"
+PLG_SYSTEM_STATS_LABEL_MESSAGE_TITLE="Helfen, Joomla! besser zu machen."
 PLG_SYSTEM_STATS_LABEL_PHP_VERSION="PHP-Version"
 PLG_SYSTEM_STATS_LABEL_SERVER_OS="Server OS"
 PLG_SYSTEM_STATS_LABEL_UNIQUE_ID="Eindeutige ID"
 PLG_SYSTEM_STATS_MODE_LABEL="Modus"
 PLG_SYSTEM_STATS_MODE_OPTION_ALWAYS_SEND="Immer senden"
 PLG_SYSTEM_STATS_MODE_OPTION_NEVER_SEND="Niemals senden"
-PLG_SYSTEM_STATS_MODE_OPTION_ON_DEMAND="Auf Aufforderung"
+PLG_SYSTEM_STATS_MODE_OPTION_ON_DEMAND="Nochmal fragen"
 PLG_SYSTEM_STATS_MSG_ALLOW_SENDING_DATA="Joomla!-Statistikerhebung aktivieren?"
-PLG_SYSTEM_STATS_MSG_JOOMLA_WANTS_TO_SEND_DATA="Das Joomla!-Projekt würde gerne die Installationsbasis und die Endbenutzerumgebungen besser kennen, damit gezielt hierfür entwickelt werden kann. Dafür wurde dieses Plugin entwickelt, um die minimal benötigten Daten an einen zentralen vom Joomla!-Projekt kontrollierten Server senden zu können. Dabei werden zu <strong>keiner Zeit</strong> persönlichen Daten ermittelt. Die getroffenen Einstellungen können später unter „Erweiterungen“ &rarr; „Plugins“ &rarr; „System - Joomla!-Statistikerhebung“ geändert werden."
-PLG_SYSTEM_STATS_MSG_WHAT_DATA_WILL_BE_SENT="Hier klicken um die zu sendenden Daten anzusehen."
+PLG_SYSTEM_STATS_MSG_JOOMLA_WANTS_TO_SEND_DATA="Das Joomla!-Projekt würde gerne die Kompatibilität steigern und benötigt dafür Informationen zur Installationsbasis und den Endbenutzerumgebungen. Alle Daten werden anonymisiert und an einen vom Joomla!-Projekt zentral gesteuerten Server übermittelt. Die Einstellungen können später unter „Verwalten“ &rarr; „Plugins“ &rarr; „System - Joomla!-Statistikerhebung“ geändert werden."
+PLG_SYSTEM_STATS_MSG_WHAT_DATA_WILL_BE_SENT="Es werden dabei zu keinem Zeitpunkt persönliche Daten ermittelt. Hier klicken, um die Daten anzusehen."
```

Comment:
```suggestion
PLG_SYSTEM_STATS_MSG_WHAT_DATA_WILL_BE_SENT="Hier klicken, um die Daten anzusehen welche Übertragen werden. Zusätzich dazu wird aus technischen Gründen die IP Adresse des Servers übertragen welche aber nicht ausgewertet wird."
```

Das ist eine falsche aussage da zumindest in deutschland die IP als Persönliche Daten gilt. ggfs ist es so klarer.

Reply:
d.h., dass diese Ansicht gar nicht alle Daten, enthält oder weil es technisch einfach so ist, eh klar ist, dass die IP übermittelt wird?

<img width="593" alt="Bildschirmfoto 2022-01-15 um 21 02 42" src="https://user-images.githubusercontent.com/1752513/149636161-27c1a1d6-8830-449a-9650-38e357fede2f.png">

Reply:
@zero-24 see https://github.com/joomlagerman/joomla/pull/2227/commits/5d5925e461f4994ba98eb5594070ff0b62ce0492